flow360/user_config.py

```diff
@@ -124,5 +131,22 @@ def enable_validation(self):
         """enable user side validation (pydantic)"""
         self._do_validation = True
 
+    @property
+    def use_system_certs(self) -> bool:
+        """retrieve the current value of the use_system_certs configuration option
+
+        Returns
+        -------
+        bool
+            whether to use the OS system certificate store for SSL validation"""
+        self._check_env_profile()
+        self._check_env_usessytemcerts()
+        if self._use_system_certs is not None:
+            return self._use_system_certs
+
+        config_map = self.config.get(self.profile, {})
+        setting = config_map.get("usesystemcerts", False)
```

Comment:
this one to work requires usesystemcerts being available in config.toml file which is managed by flow360 CLI. You can add this option to the CLI so the user can set it only once: https://github.com/flexcompute/Flow360/blob/develop/flow360/cli/app.py